apidoc/Titanium/UI/iPhone/AlertDialogStyle.yml

```diff
@@ -9,7 +9,7 @@ platforms: [iphone, ipad]
 createable: false
 deprecated:
     since: "6.0.0"
-    notes: Use [Titanium.UI.iOS.AlertDialogStyle](Titanium.UI.iOS.AlertDialogStyle) instead.
+    notes: Use Titanium.UI.iOS.AlertDialogStyle instead.
```

Comment:
We are having problems crossing linking if people use the short-hand of Titanium (Ti). Right now, our document system doesn't treat Titanium.whatever and Ti.whatever the same. Some links are broken, in particular those using the incorrect module prefix. Ultimately, we need to unify which one we are going to use.